.circleci/test_modules.txt

```diff
@@ -13,3 +13,4 @@ permissions:testDebug
 settings:testDebug
 maps:testDebug
 errors:testDebug
+crash-handler:testDebug
```

Comment:
Yeah, I had thought you couldn't do it, but it seems totally fine! Definitely nicer. We can look at renaming other modules at some point.

Reply:
What about using the underscore sign? We have it in `collect_app` module. It would be better to use one convention across the app.

Reply:
I really prefer `-`, but I don't want to rename `collect_app` 😂. It would be nice for that module to just be `app` (as is the convention). I could just rename this module `crash` so we can punt on that right now?

Reply:
You can leave crash-handler but please file a separate issue to rename other modules.

Reply:
Done: https://github.com/getodk/collect/issues/5337